pkg/controller/imagebuild/imagebuild.go

```diff
@@ -15,22 +15,35 @@ import (
 	"github.com/dominodatalab/hephaestus/pkg/controller/imagebuild/predicate"
 )
 
-var ch = make(chan client.ObjectKey)
-
-func Register(mgr ctrl.Manager, cfg config.Controller, pool worker.Pool, nr *newrelic.Application) error {
-	return core.NewReconciler(mgr).
+func Register(mgr ctrl.Manager, cfg config.Controller, pool worker.Pool, nr *newrelic.Application,
+	deleteChan chan client.ObjectKey,
+) error {
```

Comment:
Very nitpicky, but this formatting situation feels wrong

Reply:
There isn't a good way...  I can go with a long line but then the linter complains... let me see if I can force this to work well with gofumpt. But the pipelines does a gofmt so its a sad state of affairs.

Reply:
made it consistent with arg per line.